cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -507,7 +524,23 @@ teeForge ft tverb tr = Tracer $ \ev -> do
       Consensus.TraceBlockFromFuture{} -> teeForge' (ftTraceBlockFromFuture ft)
       Consensus.TraceSlotIsImmutable{} -> teeForge' (ftTraceSlotIsImmutable ft)
       Consensus.TraceNodeIsLeader{} -> teeForge' (ftTraceNodeIsLeader ft)
-  traceWith (toLogObject' tverb tr) ev
+  case event of
+    Consensus.TraceStartLeadershipCheck slot -> do
```

Comment:
Surely we can move `Consensus.TraceStartLeadershipCheck` to the above case expression?

Reply:
No, not quite, these are traced to different tracers.

Reply:
Ok, why not update `teeForge'` here then: 
https://github.com/input-output-hk/cardano-node/blob/657f2f660991b97084e8fbd38e92199f6faec8e6/cardano-node/src/Cardano/Tracing/Tracers.hs#L557